lib/brains/views/webhook.py

```diff
@@ -0,0 +1,31 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from lib.brains.client import get_client
+from lib.brains.forms import WebhookParseForm, WebhookVerifyForm
+from solitude.errors import FormError
+from solitude.logger import getLogger
+
+log = getLogger('s.brains')
+
+
+@api_view(['POST'])
+def parse(request):
+    form = WebhookParseForm(request.DATA)
+    if not form.is_valid():
+        raise FormError(form.errors)
+
+    parsed = get_client().WebhookNotification.parse(*form.braintree_data)
+    log.info('Received webhook: {p.kind}.'.format(p=parsed))
+    # TODO: actually do something with the web hook.
+    return Response(status=204)
+
+
+@api_view(['GET'])
+def verify(request):
+    form = WebhookVerifyForm(request.QUERY_PARAMS)
+    if not form.is_valid():
+        raise FormError(form.errors)
+
+    res = get_client().WebhookNotification.verify(form.braintree_data)
```

Comment:
Since we will have no visibility into this response (only Braintree does), I think you should log it.